source/tests-integration/TestFixture.ts

```diff
@@ -38,13 +38,13 @@ export class TestFixture {
 
     public async approveCentralAuthority(): Promise<void> {
         const authority = this.contractDeployer.getContract('Augur');
-        const cash = <Cash>this.contractDeployer.getContract('Cash');
+        const cash = new Cash(this.connector, this.accountManager, this.contractDeployer.getContract('Cash').address, this.configuration.gasPrice);
         const transactionHash = await cash.approve(authority.address, new BN(2).pow(new BN(256)).sub(new BN(1)));
         await this.connector.waitForTransactionReceipt(transactionHash, `Approving central authority.`);
     }
 
     public async createMarket(universe: Universe, numOutcomes: number, endTime: number, feePerEthInWei: number, denominationToken: string, designatedReporter: string, numTicks: number): Promise<Market> {
-        const legacyReputationToken = <LegacyReputationToken>this.contractDeployer.getContract('LegacyReputationToken');
+        const legacyReputationToken = new LegacyReputationToken(this.connector, this.accountManager, this.contractDeployer.getContract('LegacyReputationToken').address, this.configuration.gasPrice);
```

Comment:
I was thinking that we should make a `getContract` overload that is generic and somehow returns the right class:
```
const cash = factory<Cash>('Cash');
```
The `factory` would curry all of the other bits like connector, accountManager, contractDeployer, etc. and it would get the contract from `contractDeployer.getContract` and then instantiate the contract via a mapping of `string => constructorFunction` that is auto-generated by the code generator.

This is out of scope of this PR, but as we start authoring more and more tools/tests against the TypeScript I believe such a feature would be very useful.